constraints.txt

```
@@ -5,3 +5,6 @@ PyYAML>=6.0.1,<6.1
 # to avoid version conflicts
 requests==2.25.1
 requests-retry-session>=0.1,<0.2
+kubernetes==26.1.0
+python-keycloak==0.27.1
+urllib3==1.26.20
```

For the packages we are using from the system Python environment, if we are going to specify a version here, it should match the version installed in the CSM 1.6 images.
```suggestion
urllib3==1.25.10
```

Is there a reason that version will not work with your tests? If so, we'll need to stop using the system version of that package.